tox.ini

```diff
@@ -59,7 +59,9 @@ deps =
     pytest-cov
     pytest-xdist
 changedir = tests
-extras = soap-bpnn  # this model is used in the package tests
+extras =  # architectures used in the package tests
+    soap-bpnn
+    pet
```

Comment:
Is PET actually used in the package tests? I can't find it

Reply:
Yeah, I am also not happy, maybe one can also monkeypatch this...

Reply:
Don't worry, we should be able to take it out soon-ish